Repositories/Eloquent/EloquentBaseRepository.php

```diff
@@ -129,6 +129,30 @@ public function findByAttributes(array $attributes)
     }
 
     /**
+     * Get resources by an array of attributes
+     * @param  array  $attributes
+     * @return objects
+     */
+    public function getByAttributes(array $attributes, $orderBy = '', $sort = 'asc')
+    {
+        $query = $this->model->query();
+
+        if (method_exists($this->model, 'translations')) {
+            $query = $query->with('translations');
+        }
+
+        foreach ($attributes as $field => $value) {
+            $query = $query->where($field, $value);
+        }
```

Comment:
You can refactor everything that this method has in common with `findByAttributes` (line 116). Save some code. :)